elftools/elf/structs.py

```diff
@@ -43,9 +43,33 @@ def __init__(self, little_endian=True, elfclass=32):
         assert elfclass == 32 or elfclass == 64
         self.little_endian = little_endian
         self.elfclass = elfclass
-        self._create_structs()
 
-    def _create_structs(self):
+    def create_basic_structs(self):
```

Comment:
I'm not sure the split to _create_scalar_structs really helps and "scalar" is confusing in this context (at least to me). I suggest just inline it in here: create_basic_structs will set up all the word size stuff and call _create_ehdr.